vpn-router-alice/src/main/java/vpnrouter/alice/infrastructure/controller/Audio.java

```diff
@@ -0,0 +1,6 @@
+package vpnrouter.alice.infrastructure.controller;
+
+public class Audio {
+    public static final String SUCCESS = "<speaker audio=\"alice-music-horn-1.opus\">";
+    public static final String FAILURE = "<speaker audio=\"alice-sounds-things-toilet-1.opus\">";
```

Comment:
Where has you found this sound? 😁 There is no such sound on the official [Alice's page](https://yandex.ru/dev/dialogs/alice/doc/ru/sounds/music) =)

Reply:
:)
https://yandex.ru/dev/dialogs/alice/doc/ru/sounds